XcodeAutoBasher/XcodeAutoBasher.xcodeproj/project.pbxproj

```diff
@@ -348,6 +349,7 @@
 				GCC_PREPROCESSOR_DEFINITIONS = "$(inherited)";
 				INFOPLIST_FILE = "XcodeAutoBasher/XcodeAutoBasher-Info.plist";
 				INSTALL_PATH = "/Library/Application Support/Developer/Shared/Xcode/Plug-ins";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.vokalinteractive.${PRODUCT_NAME:rfc1034identifier}";
```

Comment:
In theory, but I think it's worth leaving be so that it doesn't try to install as a different instance via alcatraz.